### ISSUE TITLE
White ball keeps rotate when fall in hole.

### ISSUE THREAD

**Issue by MrDummyNL** [FairlySadPanda/vrcbce#121]
With the Metal version table, the white ball keeps rotate forever once it fell in the hole.
Movements are still correct. But rotation is not reset properly when respawned.

Just so weird effect.
Normal table is possible not affected (not noticed it so far)

**Comment by Metamaniac** [comment]
Yeah, did you do a clean install updating to 1.5.0? (are you ON 1.5.0??)

**Comment by Legoman99573** [comment]
re-drag in the prefab and remove the old one and backup any configuration changes you made and see if it works

**Comment by Codel1417** [comment]
Can confirm this happens. Appears angular velocity is still handled by table sim when ball is pocketed
